src/Core/Grid/Definition/Factory/AddressGridDefinitionFactory.php

```diff
@@ -290,6 +291,11 @@ protected function getBulkActions(): BulkActionCollectionInterface
                     ->setOptions([
                         'submit_route' => 'admin_addresses_delete_bulk',
                         'confirm_message' => $this->trans('Delete selected items?', [], 'Admin.Notifications.Warning'),
+                        'modal_options' => new ModalOptions([
+                            'title' => $this->trans('Delete selected addresses?', [], 'Admin.Actions'),
+                            'confirm_button_label' => $this->trans('Delete', [], 'Admin.Actions'),
+                            'confirm_button_class' => 'btn-danger',
```

Comment:
Hi @LouiseBonnard , I'm not sure about what to put here in place of "Delete selected addresses?", does it seem good to you?

Reply:
I did what we saied @LouiseBonnard